src/asciidoc/docs/publish/versioning.adoc

```diff
@@ -0,0 +1,129 @@
+= Managing template versions
+:page-slug: template-versions
+:page-description: How to keep your Koji template up to date.
+
+After you publish a template on Koji, you might want to update it to increase its value.
+For example, you might add new features or fix bugs reported by testers or remixers.
+
+The Koji platform provides several options to help you manage the update and maintenance strategy for your templates.
+You can update the template code and republish the project, which replaces the existing template with a new version.
+Alternatively, you can decide to fork the code base and publish it as a new template.
+If you update a template to a new version, you can decide how to handle instant remixes of Kojis that are based on older versions.
+
+== Updating vs. forking a template
+
+When you're planning to make changes to your template, you must decide whether to update it to a new version or fork it and create a new template.
+There are several factors to consider, including the scope of the changes and the compatibility of the Visual Customization Control (VCC) data.
+
+To update a template to a new version, you modify the code and republish the project.
+The new version replaces the older version at the same URL.
+A creator who remixes your template from the base URL can now take advantage of its latest features and fixes.
+
+However, a creator can still remix a Koji that was created from an older version of your template.
+To handle this case, you can configure the project settings so that <<_managing_instant_remix_versions,instant remixes always use the latest version>> of the template.
+For the remix experience to work the way the user expects, the template must be able to support VCC values in the older format, and any new features must have reasonable default values.
+
+If an updated template would not be able to support VCC values in the older format, or if the changes would fundamentally change how the template is used, you can consider forking the code base and creating a new template.
```

Comment:
> If an updated template would not be able to support VCC values in the older format

It feels like this might be a bit confusing given the section below on "Transforming VCC values" - maybe something more like "if an updated template introduces new VCCs or uses VCC data in different ways..."? Or maybe we could introduce the idea of transforming values earlier on this page, so the developer can understand that they can "upgrade" to resolve breaking changes between versions, but that there might be cases where they change VCC structure so fundamentally that value transformation can't work and they need to fork.

Reply:
Great feedback! I really like the idea of introducing the transformation "concept" here. I made some edits to try to make this section a bit clearer. Let me know if you think of a better way to phrase it.